python/tskit/metadata.py

```diff
@@ -39,6 +41,8 @@
 import tskit
 import tskit.exceptions as exceptions
 
+__setattr__ = builtins.object.__setattr__
```

Comment:
I wonder about the wisdom of this - don't we then export this into our namespace? If we're going to do something this ugly, the I'd prefer to feel the full brunt of the ugliness each time we use it, just to be reminded of how ugly it is.

Reply:
I imported it to avoid the double lookup on each row access - I thinking importing it to something called `__builtin__object__setattr__` would be ugly enough?